src/olympia/devhub/templates/devhub/email/revoked-key-email.ltxt

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}{% blocktrans %}The API Key was revoked from your account:
+
+{{ key }}
+
+If you believe this key was revoked in error, you can create a new one
+at the following location:
+
+{{ url('devhub.api_key')|absolutify }}
+{% endblocktrans %}
```

Comment:
The config related to what template is interpreted as a `jinja2` template or not:

```python
JINGO_EXCLUDE_PATHS = (
    'users/email',
    'reviews/emails',
    'editors/emails',
    'amo/emails',
)
```

So if you either add `devhub/email` to that list or move your file around a bit it should be interpreted as a django template then again. Maybe that's what you want (since most of our email templates are django templates)